docs/src/main/sphinx/security/ldap.rst

```diff
@@ -55,10 +55,11 @@ Property                                                      Description
                                                               Should be set to ``true``. Default value is
                                                               ``false``.
 ``http-server.https.port``                                    HTTPS server port.
-``http-server.https.keystore.path``                           The location of the Java Keystore file that will be
-                                                              used to secure TLS.
-``http-server.https.keystore.key``                            The password for the keystore. This must match the
-                                                              password you specified when creating the keystore.
+``http-server.https.keystore.path``                           The location of the PEM or Java keystore that
```

Comment:
What is our convention for these?

Would it be better to say

The location of the PEM or JKS keystore file that is used enabling TLS.

Reply:
OK